examples/connectivity/plot_adhd_covariance.py

```diff
@@ -95,13 +98,24 @@ def plot_matrices(cov, prec, title):
 
 # Displaying results ##########################################################
 print("-- Displaying results")
-title = ("Subject {0:d} GroupSparseCovariance "
-         "$\\alpha={1:.2e}$").format(plotted_subject, gsc.alpha_)
+atlas_imgs = image.iter_img(msdl_atlas_dataset.maps)
+atlas_region_coords = np.array(
+    [plotting.find_xyz_cut_coords(img) for img in atlas_imgs])
+
+title = "Subject {0:d} GroupSparseCovariance $\\alpha={1:.2e}$".format(
```

Comment:
> My subjective feeling: we mostly use the older formatted printing with %.
> Should this be (roughly) consistent across examples?

I think that we should use 'format'. It's the new recommended way of
doing things. I am myself not doing it out of habit, because I started
doing Python before it existed, but that's a bad thing.